index.js

```diff
@@ -2,89 +2,96 @@
 
```

Comment:
Hm. By using your linter settings yo removed all the `;` but I really like them :-(

Reply:
Check please